spec/abci/abci.md

```diff
@@ -610,15 +621,15 @@ via light client.
 
 ## Data Types
 
-The data types not listed below are the same as the [core data structures](../core/data_structures.md). The ones listed below have specific changes to better accommodate applications.
+The data types not listed below are the same as the [core data structures](../spec/core/data_structures.md). The ones listed below have specific changes to better accommodate applications.
```

Comment:
This seems confusing, I'm not sure how the reader is expected to know which types are supposed to be here vs. not here ("anyone who is absent, please raise your hand"). 

What I _think_ this is saying is something like:

> Most of the data structures used in ABCI are shared [common data structures](../spec/core/data_structures.md) with Tendermint Core. In certain cases, however, ABCI uses different structures, which are documented here: